.github/workflows/tests-coverage.yml

```diff
@@ -0,0 +1,33 @@
+# This workflow will do a clean installation of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-nodejs
+
+name: Node.js CI
+
+on:
+  push:
+    branches: ['*']
+  pull_request:
+    branches: ['*']
```

Comment:
Will this make Github actions run twice for single pr
Like once when PR is raised, and one for push?

Reply:
yes, it will check for both the cases, I think we can just run the tests when a `PR` is raised. wdyt?

Reply:
I have reviewed this PR a while back but never requested for change..... Was wondering why was it showing pending😅